wandb/sdk/wandb_login.py

```diff
@@ -78,6 +79,8 @@ def login(
         UsageError - if api_key cannot be configured and no tty
     """
     _handle_host_wandb_setting(host)
+    if os.environ.get(LAUNCH):
```

Comment:
could you add some comment this won't be obvious to any other dev reading this code?